src/statsig/types.ts

```diff
@@ -10,6 +9,11 @@ export enum StatsigDynamicConfigs {
   EARN_STABLECOIN_CONFIG = 'earn_stablecoin_config',
 }
 
+// Separating into different enum from StatsigDynamicConfigs to allow for more strict typing
```

Comment:
you can also use `Exclude<StatsigDynamicConfigs, StatsigDynamicConfigs.MULTI_CHAIN_FEATURES>` instead of explicitly defining 2 enums

Reply:
Exclude makes things a bit weirder because the output is a type not an enum